lib/seattleflu/id3c/cli/command/etl/redcap_det_uw_reopening.py

```diff
@@ -99,6 +106,8 @@ def redcap_det_uw_reopening(*, db: DatabaseSession, cache: TTLCache, det: dict,
 
     record_id = redcap_record_instances[0]['record_id']
     project_id = redcap_record_instances[0].project.id
+    husky_project = next((p for p in PROJECTS if p.id == project_id))
+
```

Comment:
This lookup suggests to me that `PROJECTS` would be better off as a dictionary.

Reply:
We don't need this code anymore, so I'm removed it.